jujugui/static/gui/src/app/components/service-overview/test-service-overview.js

```diff
@@ -61,6 +61,14 @@ describe('ServiceOverview', function() {
     }
   }
 
+  function getUnitStatusCounts(error=0, pending=0, uncommitted=0) {
```

Comment:
Nice. Good to see this bit pulled out into a standalone function, rather than copied-n-pasted throughout the test code.